cli/test/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--timeout 0
+--slow 90000
```

Comment:
Do you need those if you specify them when you run the tests in through the `npm run test` command?